Formula/gdub.rb

```diff
@@ -6,18 +6,14 @@ class Gdub < Formula
 
   bottle :unneeded
 
-  depends_on "gradle"
+  depends_on "gradle" => :optional
```

Comment:
Just drop this dep entirely then. It's at most a runtime dep (should be `=> :run` to begin with), and people can always install it separately if they want to.